arbitrary-file-write/file-create/README.md

```diff
@@ -49,4 +49,21 @@ asdas
 
 [we]! can have symbols and garbage at the end
 <00><ff> binary also ok here
-```
\ No newline at end of file
+```
+### Shell Profiles
+#### Login Shells
+`--login` flag. 
+* `/etc/profile` executed for all users
+* First found of `~/.bash_profile`, `~/.bash_login`, and `~/.profile` is then executed
+#### Non-login shell
+* `/etc/bash.bashrc` executed for all users 
+* `~/.bashrc` for user specific.  
+
+### Eicar antimalware testfile
+You can try to obtain it [here](https://www.eicar.org/download-anti-malware-testfile/). The most it'll do is trigger deletion by antivirus especially those with real time detection but an be helpful for triggering weird errors in some applications.  
+
+### git hooks
+One may be able to sneak in a [git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks#_git_hooks) in the `.git` of a git repo to execute any code of their choosing when a git action happens, however this requires the file to marked executable. 
```

Comment:
good idea, definitely interesting for a server as well, where a deploy script might execute a git pull. But again, we need actual examples, limitations, messy files, etc